kw

```diff
@@ -255,6 +255,13 @@ function kw()
         env_main "$@"
       )
       ;;
+    env-show)
+      (
+        include "${KW_LIB_DIR}/kw_env_show.sh"
+
+        env_show_main "$@"
+      )
+      ;;
```

Comment:
This should be configured during the setup time. Maybe you can create a new option for the `setup.sh` script.